frontend/src/cljs/imas_seamap/map/utils.cljs

```diff
@@ -198,19 +198,22 @@
         str)))
 
 (def feature-info-none
-  {:info (str
-          "<div>"
-          "<h4>No info available</h4>"
-          "Layer summary not configured"
-          "</div>")})
+  {:style nil
+   :body
+   (str
+    "<div>"
+    "    <h4>No info available</h4>"
+    "    Layer summary not configured"
+    "</div>")})
 
 (defn feature-info-html
   [response]
   (let [parsed (.parseFromString (js/DOMParser.) response "text/html")
-        body  (first (array-seq (.querySelectorAll parsed "body")))]
-    (if (.-firstElementChild body)
-      {:info (.-innerHTML body)}
-      {:status :feature-info/empty})))
+        body  (first (array-seq (.querySelectorAll parsed "body")))
+        style  (first (array-seq (.querySelectorAll parsed "style")))] ; only grabs the first style element
```

Comment:
Yeah I was going to look for this -- I suppose in the general instance it's not ideal, but for basic GFI responses that probably just have a single style block it's probably fine.

Reply:
Yep. If it comes down to it I can always right extra code to merge the contents of all style elements (but this was the simplest solution).